runtime/near-vm-runner/src/cache.rs

```diff
@@ -56,7 +56,7 @@ pub fn get_contract_cache_key(
 ) -> CryptoHash {
     let _span = tracing::debug_span!(target: "vm", "get_key").entered();
     let key = ContractCacheKey::Version3 {
-        code_hash: code.hash,
+        code_hash: code.hash().clone(),
```

Comment:
Crypto hash is copy, so here and in all other places it's better to spell `*code.hash()`. This makes it clear to the reader that no expensive allocationg clone is happening.

Reply:
fixed!